packages/asset/contracts/AssetCreate.sol

```diff
@@ -55,6 +55,18 @@ contract AssetCreate is
     /// @notice The marketplace exchange contract to purchase catalyst
     IExchange private exchangeContract;
 
+    /// @notice Maximum BPS value
+    uint256 public constant MAX_BPS_IN_UNIT = 10000;
+    /// @notice Revealed status nonce value
+    uint8 public constant REVEALED_NONCE = 1;
```

Comment:
Terminology: not technically a nonce (same for NOT_REVEALED_NONCE); would suggest IS_REVEALED, for example